src/wrap-store/wrapStore.js

```diff
@@ -99,7 +99,9 @@ export default (store, {
   /**
    * Setup external action handler
    */
-  chrome.runtime.onMessageExternal.addListener(dispatchResponse);
+  if (chrome.runtime.onMessageExternal) {
```

Comment:
Onion-layering. I agree that's the best. We can leave it here as a utility so we can play with it and test with `react-chrome-redux` and then create a separate lib as it gets some field-testing :-)